projects/1.19/assets/beachparty/beachparty/lang/zh_cn.json

```
@@ -0,0 +1,131 @@
+{
+  "beachparty.container.cabinet": "橱柜",
+  "block.beachparty.beach_chair":"沙滩摇椅",
+  "block.beachparty.beach_grass": "沙草",
+  "block.beachparty.beach_towel": "沙滩毛巾",
+  "block.beachparty.cabinet": "棕榈橱柜",
+  "block.beachparty.chair":"棕榈布艺椅子",
+  "block.beachparty.cocoa_cocktail": "§a绿野仙踪",
+  "block.beachparty.coconut": "椰子",
+  "block.beachparty.coconut_block": "椰子",
+  "block.beachparty.coconut_cocktail": "§b椰岛之恋",
+  "block.beachparty.deck_chair": "沙滩日光椅",
+  "block.beachparty.dried_wheat_block": "草垛块",
+  "block.beachparty.dried_wheat_slab": "草垛台阶",
+  "block.beachparty.dried_wheat_stairs": "草垛楼梯",
+  "block.beachparty.dry_bush":"沙灌木",
+  "block.beachparty.dry_bush_tall": "高沙灌木",
+  "block.beachparty.empty_sand_bucket": "空沙桶",
+  "block.beachparty.empty_sand_bucket_block": "空沙桶",
+  "block.beachparty.hammock": "沙滩吊床",
+  "block.beachparty.honey_cocktail": "§e淘金热",
+  "block.beachparty.lounge_chair":"沙滩躺椅",
+  "block.beachparty.melon_cocktail": "§d幸运星",
+  "block.beachparty.message_in_a_bottle": "漂流瓶",
+  "block.beachparty.mini_fridge": "迷你冰箱",
+  "block.beachparty.notimplemented" : "（这个方块有尚未完全实现的功能。请继续关注!）",
+  "block.beachparty.palm_beam": "棕榈木柱",
+  "block.beachparty.palm_button": "棕榈木按钮",
+  "block.beachparty.palm_door": "棕榈木门",
+  "block.beachparty.palm_fence": "棕榈木栅栏",
+  "block.beachparty.palm_fence_gate": "棕榈木栅栏门",
+  "block.beachparty.palm_floorboard": "雕纹棕榈木板",

```

```suggestion
  "block.beachparty.palm_floorboard": "棕榈木地板",
```
`雕纹`对应的是`chiseled`（1.19.3及以上）